test/fixtures/coverage/functional_test/lib/index.dart

```diff
@@ -1,11 +0,0 @@
-import 'dart:html';
```

Comment:
The next few files show up as completed removed, but that's only because I changed a few things and renamed the parent `test/fixtures/` directory. Most of this is the same or similar